core/src/dird/ua_tree.cc

```diff
@@ -1001,60 +987,61 @@ static int DotPwdcmd(UaContext* ua, TreeContext* tree)
   return 1;
 }
 
-static int Unmarkcmd(UaContext* ua, TreeContext* tree)
+int UnmarkElement(char* element, UaContext* ua, TreeContext* tree)
 {
-  POOLMEM* cwd;
+  int count = 0;
+  StripTrailingSlash(element);
   TREE_NODE* node;
+  // See if this is a full path.
+  if (strchr(element, '/')) {
+    int pnl, fnl;
```

Comment:
Do you know why this is the other way around here ? In `MarkElement` its
```
if (strstr(element, '/')) {
   StripTrailingSlash(element);
   ...
}
```
If you cannot find a difference, I would make the same change as above.

Reply:
MarkElement now works for both, so mark and unmark behave the same in terms of path and filename manipulation